web/client/plugins/manager/Manager.jsx

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
This has been done because we have different plugins specific for manager, so we should group all them together in that folder.
There are many plugins now, and I suggest to group them somehow.